src/services/usageTracking/index.ts

```diff
@@ -0,0 +1,108 @@
+import { app } from 'electron'
+import log from 'electron-log/main'
+import { writeFile, readFile } from 'fs/promises'
+
+import { TRACKING_URL, INSTALLATION_ID_FILE } from '@/constants/usage'
+import { getArch, getPlatform } from '@/utils/electron'
+import { uuid } from '@/utils/uuid'
+
+import {
+  UsageEvent,
+  UsageEventMetadata,
+  UsageEventName,
+  UsageEventWithMetadata,
+} from './types'
+
+export async function initEventTracking() {
+  if (process.env.NODE_ENV === 'development') {
+    return
+  }
```

Comment:
I had to comment this out to get the event to be logged to console. Maybe it's not necessary since we block it in `sendEvent`?

Reply:
The logic is pretty much the same as before - If the installation id file already exists, we won't report the installation event, so this early return prevents generation of this file.

We could instead have a separate file for the dev env and even use the dev version of the usage stats endpoint - what do you think?